### PR TITLE
Split of srcing

### PR DESCRIPTION
**Pull request by Demopans**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,8 +1,10 @@
 FROM nginx:stable
-RUN apt-get -y update
-RUN apt-get -y install git
+RUN apt-get -y update && apt-get -y install git
 WORKDIR /usr/src
-RUN git clone https://github.com/Ada18980/KinkyDungeonStandalone.git .
+RUN git clone -b PickSrc https://github.com/Demopans/KinkyDungeonStandalone.git .
```
Comment (Ada18980) [inline Dockerfile]: I think this should point to my git and not yours o.o

File: index.html [hunk 2]
```diff
@@ -2,132 +2,134 @@
...
+            .filter(x => !x.startsWith('Screens/Room/')); // Easy load time optimization
 
-        Click = () => {};
-        TouchStart = () => {};
-        MouseMove = () => {};
-        LoseFocus = () => {};
+        sourceScripts.splice(sourceScripts.indexOf('Screens/Character/ItemColor/ItemColor.js'), 0, 'src/common-shim.js');
+        sourceScripts.push('src/index-shim.js', 'src/shim.js');
 
-        load().then(
```
Comment (Ada18980) [inline index.html:118]: Won't removing this cause problems? Am not good web dev sorry :S

File: Dockerfile (hunk 1 above)
Comment (Demopans) [reply Dockerfile]: I thought I reverted that...

Comment (Demopans) [reply Dockerfile]: Oh wait, I was doing that so stuff builds on my end

File: index.html (hunk 2 above)
Comment (Demopans) [reply index.html:118]: That was from me moving a few lines to another file

**Comment by Demopans** [comment]
Ok, thing reverted to your fork. I'll be passing my version around for the time being though

**Pull request merged**